package.json

```diff
@@ -46,5 +46,10 @@
   },
   "lint-staged": {
     "*.{js,css,md,ts,tsx,json,yaml,yml}": "prettier --write"
+  },
+  "resolutions": {
+    "@walletconnect/client": "2.0.0-beta.20.3",
+    "@walletconnect/types": "2.0.0-beta.20.3",
+    "@walletconnect/utils": "2.0.0-beta.20.3"
```

Comment:
IIUC this will force the transitive deps of the v1 libs to use the v2 versions, which shouldn't work. I wouldn't even expect it to build so if it's building maybe I'm missing something. 
Note, Pedro is working on a v1+v2 shared lib that will help here but not sure about the ETA for that